src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -537,6 +541,12 @@ func (iterator *ShelfIterator) Next() (*google_example_library_v1.Shelf, error)
     return result, nil
 }
 
+// SetPageSize sets the maximum size of the next page to be
+// retrieved.
+func (iterator *ShelfIterator) SetPageSize(int pageSize) {
```

Comment:
Yes.

On Wed, May 25, 2016 at 1:18 PM Sai Cheemalapati <notifications@github.com>
wrote:

> In src/test/java/com/google/api/codegen/testdata/go_library.baseline
> <https://github.com/googleapis/toolkit/pull/153#discussion_r64613016>:
>
> > @@ -537,6 +541,12 @@ func (iterator *ShelfIterator) Next() (*google_example_library_v1.Shelf, error)
> >      return result, nil
> >  }
> >
> > +// SetPageSize sets the maximum size of the next page to be
> > +// retrieved.
> > +func (iterator *ShelfIterator) SetPageSize(int pageSize) {
>
> Should I specify the corresponding struct types to be int32 as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/googleapis/toolkit/pull/153/files/82186645eef437e5b661c7b6ec45ed49acf6e36c#r64613016>
>